arbitrum-docs/how-arbitrum-works/02-transaction-lifecycle.mdx

```diff
@@ -1,10 +1,35 @@
-# Overview: The Lifecycle of an Arbitrum Transaction
+---
+title: Sequencing, Followed by Deterministic Execution
```

Comment:
Sidebar entry is inconsistent with the new title (they are different)

Reply:
updated